### ISSUE TITLE
[Question] Randar on version below release 1.9

### ISSUE THREAD

**Issue by konoashi** [spawnmason/randar-explanation#2]
I've been looking at this magnificent exploit closely since this paper is public and after a little bit of research in MC sources a question started to haunt me, how can we know item exact coordinates before protocol version 106 (release 1.9-pre4) since the server send 3 integers for x, y and z using floor_double method thus destroying the information of the exact position for the client since the client precision is down to a 1/32 of a block. I assume randar cannot run using block drop coordinates below 1.9 and it's necessary to find a pseudo random event with an output that the client can know with absolute precision, I am right ?

**Comment by leijurv** [comment]
You're correct, in the history that we sent to Fit we mentioned that the precision through the item drop method was insufficient before 1.9. There are other ways of measuring server-side RNG that n0p discovered, we only focused on breaking blocks because it's straightforward, obvious, easy, and works in 1.12.2.

**Comment by 0-x-2-2** [comment]
There are indeed many ways of sampling the RNG. You can use any of them that leaks enough info to the client or combine them, the fewer samples you have the harder it is to solve the problem, you don't need full accuracy like what we had but you would most likely need more samples.

**Comment by m3l0n2k13** [comment]
@konoashi Have you found the solution? What can be used instead of block drop coordinates below 1.9?
